webapp/nuxt.config.js

```diff
@@ -271,7 +271,7 @@ module.exports = {
   apollo: {
     tokenName: 'human-connection-token', // optional, default: apollo-token
     cookieAttributes: {
-      expires: 3, // optional, default: 7 (days)
+      expires: 1, // optional, default: 7 (days)
```

Comment:
> [<img alt="mattwr18" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/mattwr18) **Authored by [mattwr18](https://github.com/mattwr18)**
_<time datetime="2019-08-23T06:26:20Z" title="Friday, August 23rd 2019, 8:26:20 am +02:00">Aug 23, 2019</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -271,7 +271,7 @@ module.exports = {
   apollo: {
     tokenName: 'human-connection-token', // optional, default: apollo-token
     cookieAttributes: {
-      expires: 3, // optional, default: 7 (days)
+      expires: 1, // optional, default: 7 (days)
```


this is fixing it since it's setting the cookie to expire after the same period the jwt expires?

Reply:
> [<img alt="roschaefer" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/roschaefer) **Authored by [roschaefer](https://github.com/roschaefer)**
_<time datetime="2019-08-23T07:54:21Z" title="Friday, August 23rd 2019, 9:54:21 am +02:00">Aug 23, 2019</time>_

---


This is not fixing the `nuxtServerInit` bug at all. Just an alignment that cookie expiration and JWT expiration should align.